bpf/mock/mock_conntrack_vacant.c

```diff
@@ -0,0 +1,650 @@
+/* AUTOGENERATED FILE. DO NOT EDIT. */
```

Comment:
If these files were autogenerated, do we need to commit them into the tree or can we just generate them at test time? I'd prefer the latter so we have fewer files to manage or keep up to date or ensure developers understand how to generate them correctly.

Reply:
Oh and if we're worried about tooling not being available to generate & run the tests if the mock framework requires something unusual, the usual approach we use is just to wrap the required tooling into a docker container and mount the host filesystem into there to bootstrap.

Reply:
Make sense! I suggest we keep the generated mock library for helper functions, assuming that the list of helpers functions does not change often. We can bypass the generated mock library for customized functions, and developers can generate them upon using.